mindbender/maya/plugins/collect_instances.py

```diff
@@ -61,6 +61,11 @@ def process(self, context):
                               objectsOnly=True):    # Return objectSet, rather
                                                     # than its members
 
+            # Check if objectset is empty
+            if cmds.sets(objset, q=True) is None:
```

Comment:
Tiny nitpick, but could you change `q=True` to `query=True`? This way (1) arguments are consistent across the board and (2) it will be undeniable understandable by the next person who may not have used the `sets()` command before.